client/src/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
And this is just making it so that you can call it `Router` instead of `BrowserRouter` because the latter is a mouthful and most people have come to just doing the former 👍